.github/workflows/publish-rust-crate.yml

```diff
@@ -122,4 +125,4 @@ jobs:
         if: github.event.inputs.create_release == 'true' && github.event.inputs.dry_run != 'true'
         uses: ncipollo/release-action@v1
         with:
-          tag: rust@v${{ steps.publish.outputs.new_version }}
+          tag: ${{ steps.publish.outputs.crate_name }}@v${{ steps.publish.outputs.new_version }}
```

Comment:
nit: The crate name is a bit ugly for the tag... can we use the end of the path? Most of the other tags are using that, ie. `js` / `interface` / `program`